webplugin/js/app/km-richtext-markup-1.0.js

```diff
@@ -200,7 +200,20 @@ Kommunicate.markup = {
             </div>`
         );
     },
-
+    getLinkPreviewTemplate: function (extractedData) {
+        var data = extractedData.data;
+        return (
```

Comment:
can data be empty?

Reply:
it will return an object containing metadata of URL.